xml/storage_filesystems.xml

```diff
@@ -2026,10 +2026,10 @@ Label: none uuid: 40123456-cb2c-4678-8b3d-d014d1c78c78
    <title>Btrfs: Balancing Data across Devices</title>
    <para>
     The <command>btrfs balance</command> command is part of the
-    <package>btrfsmaintenance</package> package. It balances block groups on a
-    Btrfs file systems. For example, if you have 1 TB drive with 600 GB used by
-    data and you add another 1 TB drive, the balancing will theoretically
-    result in having 300 GB used space on each drive.
+    <package>btrfsmaintenance</package> package. It balances block groups on
+    Btrfs file systems. For example, if you have 1&nbsp;TB drive with 600&nbsp;GB used by
```

Comment:
This still shows btrfs balance a part of btrfsmaintenance. btrfs balance is a part of btrfs-progs.

Reply:
> This still shows btrfs balance a part of btrfsmaintenance. btrfs balance is a part of btrfs-progs.

This should have already been fixed. Refresh and verify please.